src/components/LocaleSwitcher/LocaleSwitcher.tsx

```diff
@@ -0,0 +1,23 @@
+import { useLocale, useTranslations } from 'next-intl';
+
+import { routing } from '../../i18n/routing';
+import LocaleSwitcherSelect from '../LocaleSwitcherSelect/LocaleSwitcherSelect';
```

Comment:
```suggestion
import { routing } from '.@/i18n/routing';
import LocaleSwitcherSelect from '../LocaleSwitcherSelect';
```
LocaleSwitcherSelect * 3